### DISCUSSION TITLE
Pytorch v1.2 整体进度（校对）

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/pytorch-doc-zh#617] [Announcements]
> 认领须知

```
提交的时候不要改动文件名称，
即使它跟章节标题不一样也不要改，
因为文件名和原文的链接是对应的！！！
```

* 留言格式: 昵称 + QQ + 章节
* 需要取消认领的也在此留言。（2个月不更新，默认放弃任务）
* 公式问题如何解决: <https://github.com/apachecn/pytorch-doc-zh/issues/360>
* 英文教程：<https://pytorch.org/tutorials/>
* 英文文档：<https://pytorch.org/docs/stable/>
* 中文教程和文档参考v1.0版本：<https://pytorch.apachecn.org/docs/1.0/>
* 模版文章: <https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/chatbot_tutorial.md>
* 知乎中文专栏： <https://zhuanlan.zhihu.com/c_1141766027527385088>
* 图片存放地址： docs/1.2/img   目录下，记得文章也改为相对路径

| 章节   | 校验者 | 进度 |
| --- | --- | --- |
| 中文教程 | - | - |
| 入门 | - | - |
| [PyTorch 深度学习: 60 分钟极速入门](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/deep_learning_60min_blitz.md) | @FoxerLee | 100% |
| [数据加载和处理教程](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/data_loading_tutorial.md) | @yangkae | 100% |
| [用例子学习 PyTorch](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/pytorch_with_examples.md) | @yangkae | 100% |
| [迁移学习教程](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/transfer_learning_tutorial.md) | @DrDavidS  | 100%  |
| [部署与TorchScript一个Seq2Seq模型](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/deploy_seq2seq_hybrid_frontend_tutorial.md) | @FoxerLee | 100%  |
| [可视化模型，数据，和与训练TensorBoard](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/intermediate/tensorboard_tutorial.md) | @jiangzhonglian  | 100% |
| [保存和加载模型](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/saving_loading_models.md) | @jiangzhonglian | 100%  |
| [torch.nn 到底是什么？](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/nn_tutorial.md) | @DrDavidS | 100% |
| 图片 | - | - |
| [TorchVision 对象检测微调教程](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/intermediate/torchvision_tutorial.md) | @jiangzhonglian  | 100% |
| [微调 TorchVision 模型](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/finetuning_torchvision_models_tutorial.md) | @jiangzhonglian | 100% |
| [空间变压器网络教程](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/intermediate/spatial_transformer_tutorial.md) | @jiangzhonglian | 100%  |
| [使用PyTorch进行神经网络传递](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/advanced/neural_style_tutorial.md) | @jiangzhonglian | 100% |
| [对抗性示例生成](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/fgsm_tutorial.md) | @jiangzhonglian | 100% |
| [DCGAN教程](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/dcgan_faces_tutorial.md) | @jiangzhonglian | 100%  |
| 音频 | - | - |
| [torchaudio教程](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/audio_preprocessing_tutorial.md) | @jiangzhonglian | 100% |
| 文本 | - | - |
| [NLP From Scratch: 使用char-RNN对姓氏进行分类](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/intermediate/char_rnn_classification_tutorial.md) | @HelWireless | 100% |
| [NLP From Scratch: 生成名称与字符级RNN](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/intermediate/char_rnn_generation_tutorial.md) | @HelWireless  | 100% |
| [NLP From Scratch: 基于注意力机制的 seq2seq 神经网络翻译](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/intermediate/seq2seq_translation_tutorial.md) | @DrDavidS | 100% |
| [文本分类与TorchText ](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/text_sentiment_ngrams_tutorial.md) | @PengboLiu  | 100% |
| [语言翻译与TorchText ](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/torchtext_translation_tutorial.md) | @PengboLiu | 100% |
| [序列对序列建模nn.Transformer和TorchText](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/transformer_tutorial.md) | @dabney777  | 100% |
| [强化学习（DQN）教程](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/intermediate/reinforcement_q_learning.md) | @wutongzhang | 100% |
| 在生产部署PyTorch模型 | - | - |
| [通过带有Flask的REST API在Python中部署PyTorch](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/intermediate/flask_rest_api_tutorial.md) | @pkwangwanjun |  |
| [介绍TorchScript](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/Intro_to_TorchScript_tutorial.md) | @HelWireless  | 100% |
| [在C ++中加载TorchScript模型](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/advanced/cpp_export.md) |  |  |
| [（可选）将模型从PyTorch导出到ONNX并使用ONNX Runtime运行](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/advanced/super_resolution_with_onnxruntime.md) |  |  |
| 并行和分布式训练 | - | - |
| [模型并行化最佳实践](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/intermediate/model_parallel_tutorial.md) | @sherlockbear | 100% |
| [分布式数据并行(DDP)入门](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/intermediate/ddp_tutorial.md) | @sherlockbear | 100% |
| [PyTorch编写分布式应用](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/intermediate/dist_tuto.md) |  |  |
| [（高级）PyTorch 1.0分布式训练与Amazon AWS](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/beginner/aws_distributed_training_tutorial.md)  | | |
| 扩展PyTorch | - | - |
| [使用自定义 C++ 扩展算TorchScript ](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/advanced/torch_script_custom_ops.md) | @FoxerLee |  |
| [用 numpy 和 scipy 创建扩展](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/advanced/numpy_extensions_tutorial.md) | @FoxerLee | 100% |
| [自定义 C++ 和CUDA扩展](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/advanced/cpp_extension.md) | @FoxerLee | 100% |
| PyTorch在其他语言 | - | - |
| [使用PyTorch C++ 前端](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/advanced/cpp_frontend.md) |  |  |
| 中文文档 | - | - |
| 注解 | - | - |
| [自动求导机制](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/notes/autograd.md) | @PengboLiu | 100% |
| [广播语义](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/notes/broadcasting.md) | @AlexJakin  | 100% |
| [CPU线程和TorchScript推理](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/notes/cpu_threading_torchscript_inference.md) | @AlexJakin  |  |
| [CUDA语义](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/notes/cuda.md) | @AlexJakin | 100% |
| [扩展PyTorch](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/notes/extending.md) |  |  |
| [常见问题](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/notes/faq.md) |  |  |
| [对于大规模部署的特点](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/notes/large_scale_deployments.md) |  |  |
| [多处理最佳实践](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/notes/multiprocessing.md) |  |  |
| [重复性](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/notes/randomness.md) |  |  |
| [序列化语义](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/notes/serialization.md) |  |  |
| [Windows 常见问题](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/notes/windows.md) |  |  |
| 社区 | - | - |
| [PyTorch贡献说明书](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/community/contribution_guide.md) |  |  |
| [PyTorch治理](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/community/governance.md) |  |  |
| [PyTorch 维护者](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/community/persons_of_interest.md) | - | 100%  |
| 封装参考文献 | - | - |
| [torch](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/torch.md) | @张假飞 @飞龙 | 100% |
| [torch.Tensor](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/tensors.md) |  |  |
| [Tensor Attributes](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/tensor_attributes.md) |  |  |
| [Type Info](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/type_info.md) |  |  |
| [torch.sparse](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/sparse.md) |  |  |
| [torch.cuda](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/cuda.md) |  |  |
| [torch.Storage](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/storage.md) |  |  |
| [torch.nn](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/nn.md) |  |  |
| [torch.nn.functional](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/nn.functional.md) |  |  |
| [torch.nn.init](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/nn.init.md) |  |  |
| [torch.optim](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/optim.md) |  |  |
| [torch.autograd](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/autograd.md) |  |  |
| [torch.distributed](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/distributed.md) |  |  |
| [torch.distributions](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/distributions.md) |  |  |
| [torch.hub](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/hub.md) |  |  |
| [torch.jit](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/jit.md) |  |  |
| [torch.multiprocessing](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/multiprocessing.md) |  |  |
| [torch.random](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/random.md) |  |  |
| [torch.utils.bottleneck](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/bottleneck.md) |  |  |
| [torch.utils.checkpoint](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/checkpoint.md) |  |  |
| [torch.utils.cpp_extension](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/cpp_extension.md) |  |  |
| [torch.utils.data](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/data.md) | @shuziP | 100% |
| [torch.utils.dlpack](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/dlpack.md) |  |  |
| [torch.utils.model_zoo](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/model_zoo.md) |  |  |
| [torch.utils.tensorboard](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/tensorboard.md) | @shuziP | 100% |
| [torch.onnx](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/onnx.md) |  |  |
| torchvision 参考文献 | - | - |
| [torchvision](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.2/torchvision/index.md) |  |  |

**Comment by dabney777** [comment]
dabney777
376817601
序列到序列与nn.Transformer和TorchText建模

**Comment by PengboLiu** [comment]
PengboLiu
565879766
文本分类与TorchText && 语言翻译与TorchText

**Comment by DrDavidS** [comment]
NimenDavid
312648004
迁移学习教程

**Comment by HelWireless** [comment]
松鼠
2587225803
NLP From Scratch: 使用char-RNN对姓氏进行分类   100%
NLP From Scratch: 生成名称与字符级RNN  100%

**Comment by wutongzhang** [comment]
梧桐
544635106
强化学习（DQN）教程

**Comment by yangkae** [comment]
yangkae
840176804
数据加载和处理教程

**Comment by DrDavidS** [comment]
DrDavidS
312648004
torch.nn 到底是什么？

**Comment by yangkae** [comment]
yangkae
840176804
用例子学习 PyTorch

**Comment by jiangzhonglian** [comment]
片刻
529815144
torchaudio教程

**Comment by PengboLiu** [comment]
PengboLiu
565879766
自动求导机制

**Comment by DrDavidS** [comment]
DrDavidS
312648004
NLP From Scratch: 用序列到序列网络和翻译注意

**Comment by mandox-yh** [comment]
zyh1204
799050408
NLP从无到有：用序列到序列网络和翻译注意

**Comment by jiangzhonglian** [comment]
> zyh1204
> 799050408
> NLP从无到有：用序列到序列网络和翻译注意

选择其他的章节吧，这个有人已经选了

**Comment by jiangzhonglian** [comment]
片刻
529815144
100% TorchVision 对象检测微调教程
100% 微调 TorchVision 模型 
100% 空间变压器网络教程	
100% 使用PyTorch进行神经网络传递
100% 对抗性示例生成
100% DCGAN教程

**Comment by casualmine** [comment]
pkwangwanjun
328518864
部署PyTorch在Python经由REST API从Flask

**Comment by FoxerLee** [comment]
foxerlee 505311321
1. PyTorch 深度学习: 60 分钟极速入门
2. 部署与TorchScript一个Seq2Seq模型

**Comment by HelWireless** [comment]
松鼠
2587225803
2.介绍TorchScript

**Comment by AlexJakin** [comment]
Alex Jakin
1272296763
 注释： 1. 广播语义 2. CPU线程和TorchScript推理

**Comment by lhc741** [comment]
lhc741
568547745
保存和加载模型

**Comment by yachenkang** [comment]
模型并行化最佳实践
123725390

**Comment by yachenkang** [comment]
分布式数据并行(DDP)入门
123725390

**Comment by shuziP** [comment]
shuzip 
123511688
校对 torch.utils.tensorboard

**Comment by FoxerLee** [comment]
使用自定义 C++ 扩展算TorchScript
~~创建扩展使用numpy的和SciPy的~~
~~自定义 C++ 和CUDA扩展~~

505311321

foxerlee

**Comment by shuziP** [comment]
shuziP
123511688
校对 torch.utils.data

**Comment by AlexJakin** [comment]
好的&nbsp;这个月事情多




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "片刻"<notifications@github.com&gt;; 
发送时间: 2019年12月2日(星期一) 上午10:07
收件人: "apachecn/pytorch-doc-zh"<pytorch-doc-zh@noreply.github.com&gt;; 
抄送: "〆"<1272296763@qq.com&gt;; "Mention"<mention@noreply.github.com&gt;; 
主题: Re: [apachecn/pytorch-doc-zh] 整体进度 v1.2（校对）  (#422)




@pkwangwanjun @AlexJakin  @lhc741  @sherlockbear 记得加速哦，哈哈哈哈

**Comment by GG-yuki** [comment]
使用PyTorch C++ 前端
qq是1390867192（封号中，三月中旬会解封）
wx是GG_yuki
如果有任何问题也可以私信联系我

**Comment by jiangzhonglian** [comment]
1.4 版本: https://github.com/apachecn/pytorch-doc-zh/issues/487
